### ISSUE TITLE
[Bug]: options lust is not Accessible with NVDA

### ISSUE THREAD

**Issue by makhlwf** [th-ch/youtube-music#3320]
### Preflight Checklist

- [x] I use the latest version of YouTube Music (Application).
- [x] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [x] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

3.9.0

### Checklists

- [ ] I use the portable version of the YouTube Music Application.
- [ ] I can reproduce this issue in the [official version of (WEB) YTM](https://music.youtube.com).

### What operating system are you using?

Windows

### Operating System Version

Windows 11 24H2 (AMD64) build 26120.3941

### What CPU architecture are you using?

x64

### Last Known Working YouTube Music (Application) version

2.2.0

### Reproduction steps

1. Turn on NVDA
2. open youtube mysic by th_ch
3 Access the options menu
4.  You can't modify the application settings e.g. Check boxes are not  Announced, and The navigation feels unresponsive not like the previous version that I have pointed to

### Expected Behavior

I expect the options menu to feel responsive is a navigation the values are pronounced and it's easy to navigate like the version 2.2.0

### Actual Behavior

The current options many feels like a plain text and most of the set values is not accessible and pronounced to screen readers like NVDA

### Enabled plugins

one
